src/app/PlanetList/PlanetList.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { Pagination, PaginationVariant, DataList, PageSection } from '@patternfly/react-core';
+import { Planet } from '../Planet/Planet';
+import axios from "axios";
+import queryString from "query-string";
+
+const PlanetList: React.FunctionComponent<any> = (props) => {
+    const [page, setPage] = useState(1);
```

Comment:
The page info is already in the url, isn't it? You shouldn't duplicate it, keeping a single source of truth is important.